src/util/segmentation/operations/fillInside.js

```diff
@@ -0,0 +1,56 @@
+import { getBoundingBoxAroundPolygon } from '../boundaries';
```

Comment:
It looks like all of my `erase`, `eraseInside`, and `eraseOutside` comments will apply to the `fill` methods, so I'll cruise through them, and only point out new items.